### PR TITLE
Added 'Thank you for contributing' in arabic.

### PR DESCRIPTION
**Pull request by AshwanthKumar7230**
- [https://github.com/AshwanthKumar7230/gci18.fossasia.org/blob/master/js/thankyou.js ] Included a Preview link and screenshot showing after and before the changes.
- [ Added 'Thank you for contributing' in arabic.] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://github.com/AshwanthKumar7230/gci18.fossasia.org/blob/master/js/thankyou.js)

- Fixes #<20>

## Description / Changes
Added 'Thank you for contributing' in arabic.

### Before:
before adding thank you for contributing in arabic

### After:
after adding thank you for contributing in arabic

- - - - - - - - - - - -

### CODE REVIEW

File: js/thankyou.js [hunk 1]
```diff
@@ -31,7 +31,7 @@
 'സംഭാവന ചെയ്തത്തിന് നന്ദി',
 'Takk fyrir þitt framlag!',
 'Tibi gratias ago pro tui contributions!'
```
Comment (nikit19) [inline js/thankyou.js]: add a comma

**Comment by AshwanthKumar7230** [comment]
Kept the comma

File: js/thankyou.js (hunk 1 above)
Comment (nikit19) [reply js/thankyou.js]: there should be a comma here. in line no. 33

**Comment by abishekvashok** [comment]
Thanks

**Comment by ammarbinfaisal** [comment]
This appears utterly weird.
I initially added "thank you contributing" in arabic. That was the first
task which I did. Who removed it? 😕

**Pull request merged**